src/common/feedGenerator.ts

```diff
@@ -222,6 +222,7 @@ export const applyFeedWhere = (
   removeHiddenPosts = true,
   removeBannedPosts = true,
   allowPrivateSources = true,
+  allowSquadPosts = true,
```

Comment:
More of a question, is there any existing flow that uses this as default currently?

Reply:
feed and bookmarks